dapptools/src/seth.rs

```diff
@@ -16,17 +16,17 @@ use structopt::StructOpt;
 async fn main() -> eyre::Result<()> {
     let opts = Opts::from_args();
     match opts.sub {
-        Subcommands::FromAscii { text } => {
-            println!("{}", SimpleSeth::from_ascii(&text));
+        Subcommands::FromUtf8 { text } => {
```

Comment:
We want the ascii aliases to be compatible with upstream! Let's have both names with the same implementation and note that one is the alias of the other?

Reply:
Upstream being the original command line tool? I added an alias for that here: https://github.com/gakonst/dapptools-rs/pull/28/files#diff-162c4eb7d539d0bf957db975fd66a58d2fff454e7bc8fca1c3f78733e02f465fR14

Or did you mean from an API/ABI point of view?

Reply:
Oh sorry my mistake, I did not see the `aliases` tag.